docs/en/week02/02-1.md

```diff
@@ -97,12 +101,12 @@ A **Gradient Based Method** is a method/algorithm that finds the minima of a fun
   $$
 
 - For SGD (Stochastic Gradient  Descent), the update rule becomes :
-  - Pick a $p$ in $\text{0...P-1}$, then update
+  - Pick a $p$ in $\text{0,\dots,P-1}$, then update
```

Comment:
Same for this one, for [`ZH`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/zh/week02/02-1.md#L152), [`ES`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/es/week02/02-1.md#L144), [`KO`](https://github.com/Atcold/pytorch-Deep-Learning/blame/b8231a30636b9552ae1980a9be6f9c3de967eb14/docs/ko/week02/02-1.md#L121), and [`IT`](https://github.com/Atcold/pytorch-Deep-Learning/blame/master/docs/it/week02/02-1.md#L83).